app/src/routes/index.ts

```diff
@@ -10,17 +10,21 @@ import { ruleRoutes } from "features/rules/routes";
 import { desktopSessionsRoutes } from "./desktopSessionRoutes";
 import { settingRoutes } from "features/settings/routes";
 import { mockServerRoutes } from "features/mocks/routes";
+import { isAppTypeSessionBear } from "utils/AppUtils";
+import { sessionBearRoutes } from "./sessionBearRoutes";
 
-export const routes: RouteObject[] = [
-  ...ruleRoutes,
-  ...sessionRoutes,
-  ...apiClientRoutes,
-  ...accountRoutes,
-  ...authRoutes,
-  ...desktopRoutes,
-  ...mockServerRoutes,
-  ...onboardingRoutes,
-  ...settingRoutes,
-  ...miscRoutes,
-  ...desktopSessionsRoutes,
-];
+export const routes: RouteObject[] = isAppTypeSessionBear()
```

Comment:
rename the original route object to `requestlyRoutes`
Export `sessionBearRoutes` also.

Then in [`src/index.jsx` Line:42](https://github.com/requestly/requestly/blob/db93c06ef8c7069165bf7ba7346c69eaf707fd8a/app/src/index.jsx#L42)

replace with `children: isAppTypeSessionBear() ? sessionBearRoutes : [...routes, ...fullScreenRoutes],`